view/app-jarvis.js

```diff
@@ -15,22 +15,10 @@ app.config(function($routeProvider) {
 
 app.controller('MainController', function($scope,$location,$rootScope,$http) {
 
-	// eslint-disable-next-line no-undef
```

Comment:
seems like you have not pulled the latest commits from the source repository. please pull the latest changes, since the PR removes many features which were later.

Either pull the latest commits, or else, copy this particular file from the present directory and directly paste the code, and then commit it.